src/annotator/highlighter.js

```diff
@@ -191,6 +214,7 @@ export function highlightRange(normedRange, cssClass = 'hypothesis-highlight') {
   // Filter out text node spans that consist only of white space. This avoids
   // inserting highlight elements in places that can only contain a restricted
   // subset of nodes such as table rows and lists.
+  const white = /^\s*$/;
```

Comment:
I realize this is inherited, but `whitespace` would be a nicer name here.

Reply:
Done